src/test/java/org/cactoos/scalar/CheckedTest.java

```diff
@@ -46,6 +49,30 @@ public void runtimeExceptionGoesOut() throws Exception {
         ).value();
     }
 
+    @Test(expected = IllegalStateException.class)
```

Comment:
@baudoliver7 please use `Assertion` and the matcher `Throws` instead of this annotation.

Reply:
@victornoel Good